src/nnet3/nnet-cudnn-simple-component.cc

```diff
@@ -294,6 +289,27 @@ void CuDNN3DConvolutionComponent::InitFromConfig(ConfigLine *cfl) {
   ok = ok && cfl->GetValue("filt-y-step", &filt_y_stride);
   ok = ok && cfl->GetValue("filt-z-step", &filt_z_stride);
 
+  // optional argument
```

Comment:
This is optional because when you load the convolutional filter from a serialized matrix on disk, we expect only a single output channel, right? If so, please comment that.